log/config.go

```diff
@@ -105,27 +105,45 @@ func init() {
 
 // prepZap is a utility function used by the Configure function.
 func prepZap(options *Options) (zapcore.Core, zapcore.Core, zapcore.WriteSyncer, error) {
-	encCfg := zapcore.EncoderConfig{
-		TimeKey:        "time",
-		LevelKey:       "level",
-		NameKey:        "scope",
-		CallerKey:      "caller",
-		MessageKey:     "msg",
-		StacktraceKey:  "stack",
-		LineEnding:     zapcore.DefaultLineEnding,
-		EncodeLevel:    zapcore.LowercaseLevelEncoder,
-		EncodeCaller:   zapcore.ShortCallerEncoder,
-		EncodeDuration: zapcore.StringDurationEncoder,
-		EncodeTime:     formatDate,
-	}
-
 	var enc zapcore.Encoder
-	if options.JSONEncoding {
+	if options.useStackdriverFormat {
+		encCfg := zapcore.EncoderConfig{
```

Comment:
can you link to the format? I assume https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry?

Reply:
done.